data/Dockerfiles/solr/solr_core/conf/solrconfig.xml

```diff
@@ -0,0 +1,1482 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Does this file contain any customizations? I didn't find it referenced in the Dovecot FTS setup guides around the internet. If it's the default, maybe you can leave it out or copy from examples.

Reply:
Version does not match, pathes could be wrong etc. I don't think it is a good idea to copy a core config as xml. We should just use the API...